modal/partial_function.py

```diff
@@ -307,10 +307,10 @@ def wrapper(raw_f: Callable[..., Any]) -> _PartialFunction:
                 "@app.function()\n@app.web_endpoint()\ndef my_webhook():\n    ..."
             )
         if not wait_for_response:
-            deprecation_warning(
+            deprecation_error(
                 (2024, 5, 13),
                 "wait_for_response=False has been deprecated on web endpoints. See "
-                + "https://modal.com/docs/guide/webhook-timeouts#polling-solutions for alternatives",
+                "https://modal.com/docs/guide/webhook-timeouts#polling-solutions for alternatives.",
             )
             _response_mode = api_pb2.WEBHOOK_ASYNC_MODE_TRIGGER
```

Comment:
this line is dead now

Reply:
Hm True — I guess i'll add a comment on what we used that for in the proto file itself.